pkg/build/builder/cmd/scmauth/sslcert.go

```diff
@@ -0,0 +1,42 @@
+package scmauth
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	SSLCertName      = "ssl-cert"
```

Comment:
We should not use ssl in code, it's an outdated term.  tls is appropriate
where clarity is needed, or Jordan's suggestion of clientCertificate.

On Aug 26, 2015, at 11:24 PM, Jordan Liggitt <notifications@github.com>
wrote:

In pkg/build/builder/cmd/scmauth/sslcert.go
<https://github.com/openshift/origin/pull/4387#discussion_r38056966>:

> @@ -0,0 +1,42 @@
> +package scmauth
> +
> +import (
> +	"fmt"
> +	"io/ioutil"
> +	"net/url"
> +	"path/filepath"
> +	"strings"
> +)
> +
> +const (
> +	SSLCertName      = "ssl-cert"

if we wanted to allow client cert auth, we'd need both the cert and key,
and would set them in git config thusly:

http.sslCert = ...
http.sslKey = ...

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4387/files#r38056966>.